--- /usr/bin/papirus-folders	2020-02-16 12:05:26.280371744 -0300
+++ papirus-folders	2020-02-24 16:40:46.235880511 -0300
@@ -119,7 +119,7 @@
 	local -a valid_colors=("black" "blue" "bluegrey" "brown" "cyan"
 		"deeporange" "green" "grey" "indigo" "magenta" "nordic"
 		"orange" "pink" "purple" "red" "teal" "violet" "white"
-		"yellow" "auroragreen")
+		"yellow" "auroragreen" "auroraorange") #Put the name of the color in this array!
 
 	for color in "${valid_colors[@]}"; do
 		colors=( "${colors[@]}" "$color" )
